app/elements/youneeq-content/youneeq-content.html

```diff
@@ -11,11 +11,13 @@
     </style>
 
     <iron-signals on-iron-signal-youneeq-suggestions="suggestionsRecieved"></iron-signals>
+    <iron-signals on-iron-signal-youneeq-callbackid="callbackIdRecieved"></iron-signals>
```

Comment:
This secondary declaration is not truly needed. Its not going to hurt anything but this could be added directly to the one above.

<iron-signals on-iron-signal-youneeq-suggestions="suggestionsRecieved" on-iron-signal-youneeq-callbackid="callbackIdRecieved"></iron-signals>